third_party/blink/renderer/modules/cobalt/crash_context/crash_context.idl

```diff
@@ -0,0 +1,23 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[
+    Exposed=(Window),
+    SecureContext
+]
+interface CrashContext {
```

Comment:
I think we want to keep the H5vcc prefix for our interfaces

Reply:
You're thinking something like `H5vccCrashContext `?

Reply:
Exactly

Reply:
Will add this as an open question to the h5vcc survey doc for Kabuki and other teams , as you suggested. I'd prefer to omit the prefix for now, if you're ok with that.